src/mixins/itext_behavior.mixin.ts

```diff
@@ -445,7 +385,7 @@ export abstract class ITextBehaviorMixin<
     }
     if (this.canvas) {
       this.canvas.calcOffset();
-      this.exitEditingOnOthers(this.canvas);
+      this.canvas.exitTextEditing();
```

Comment:
do we want to have this alias here, or since the textEditingManager is a present in every class we just call the method directly and we are explicit we are using that manager?

Reply:
Not sure I understand

Reply:
Ne already